spec/DependencyInjection/Compiler/KernelPassSpec.php

```diff
@@ -43,7 +43,7 @@ class KernelPassSpec extends ObjectBehavior
 //        $container->getDefinition('fos_http_cache.user_context.hash_generator')->willReturn($hashGenerator);
 //        $hashGenerator->getArguments()->willReturn([
 //            [
-//                $ref1 = new Reference('ezplatform.http_cache.user_context_provider.role_identify'),
+//                $ref1 = new Reference(\Ibexa\HttpCache\ContextProvider\RoleIdentify::class),
```

Comment:
Can we get rid of this commented code?

Reply:
It was commented as it should be moved (as all spec tests) to phpunit and I am almost sure that if we remove it here it will be lost forever.